napalm_logs/auth.py

```diff
@@ -124,11 +129,18 @@ def start(self):
             except ssl.SSLError:
                 log.exception('SSL error', exc_info=True)
                 continue
+            except socket.error:
+                if self.__up is False:
+                    break
+                else:
+                    raise
```

Comment:
Please let's raise and catch later the `NapalmLogsExit` exception (https://github.com/napalm-automation/napalm-logs/blob/master/napalm_logs/exceptions.py#L47) -- unused yet, but defined exactly for this scope :)